extensions/binding/org.eclipse.smarthome.binding.wemo/src/main/java/org/eclipse/smarthome/binding/wemo/handler/WemoLightHandler.java

```diff
@@ -393,13 +393,13 @@ private synchronized void onSubscription() {
     }
 
     private synchronized void removeSubscription() {
-        logger.debug("Removing WeMo GENA subscription for '{}'", this);
         if (service.isRegistered(this)) {
-            String subscription = null;
+        logger.debug("Removing WeMo GENA subscription for '{}'", this);
```

Comment:
I think the formatter did not work correctly here, there is an indentation level missing.

Reply:
Indeed, will fix it